compiler/noirc_frontend/src/parser/mod.rs

```diff
@@ -291,6 +293,7 @@ impl std::fmt::Display for SortedModule {
 #[derive(Clone, Debug, Default)]
 pub struct ParsedModule {
     pub items: Vec<Item>,
+    pub inner_attributes: Vec<SecondaryAttribute>,
```

Comment:
Hmm I was going to object to these not being in the `items` vec so that they'd keep their order when we run `nargo fmt` but I suppose we probably want to move them all to the top of the program anyway.